trinity/utils/chains.py

```diff
@@ -114,17 +122,32 @@ class ChainConfig:
 
     port: int = None
 
+    bootstrap_nodes: Tuple[Node, ...] = None
+
     def __init__(self,
                  network_id: int,
                  data_dir: str=None,
                  nodekey_path: str=None,
                  nodekey: PrivateKey=None,
                  sync_mode: str=SYNC_FULL,
-                 port: int=30303) -> None:
+                 port: int=30303,
+                 bootstrap_nodes: Tuple[Node, ...]=None) -> None:
         self.network_id = network_id
         self.sync_mode = sync_mode
         self.port = port
 
+        if bootstrap_nodes is None:
+            if self.network_id == MAINNET_NETWORK_ID:
+                self.bootstrap_nodes = tuple(
+                    Node.from_uri(enode) for enode in MAINNET_BOOTNODES
+                )
+            elif self.network_id == ROPSTEN_NETWORK_ID:
+                self.bootstrap_nodes = tuple(
+                    Node.from_uri(enode) for enode in ROPSTEN_BOOTNODES
+                )
```

Comment:
Especially since we'll see this pattern, maybe a utility method is worth it. Or maybe just:
```py
               self.bootstrap_nodes = tuple(map(Node.from_uri, ROPSTEN_BOOTNODES))
```

Reply:
I'm actually looking to add support back into at least the high level interface classes to accept either format, but I wanted to move this direction first to make things strict before loosening them back up.

Reply:
`normalize_kademlia_nodes(*node_or_enodes)`